packages/uniforms-bootstrap3/src/ListAddField.tsx

```diff
@@ -35,15 +35,28 @@ function ListAdd({
   const limitNotReached =
     !disabled && !(parent.maxCount! <= parent.value!.length);
 
+  function onAction(
+    event:
+      | React.KeyboardEvent<HTMLDivElement>
+      | React.MouseEvent<HTMLDivElement, MouseEvent>,
+  ) {
```

Comment:
As we are not using the `currentTarget` property, we can simplify it:
```suggestion
  function onAction(event: React.KeyboardEvent | React.MouseEvent) {
```

Reply:
Changed